less/core/accessibility.less

```diff
@@ -3,32 +3,33 @@
 .component { position: relative; }
 .block { position: relative; }
 
-// jquery.a11y start
-// Invisible aria-labels
-// --------------------------------------------------
-.aria-label {
-  position: absolute !important;
-  &.relative {
-    position: relative !important;
-  }
-  left: 0 !important;
-  width: auto !important;
-  height: auto !important;
-  overflow: auto !important;
-  color: rgba(0,0,0,0) !important;
-  background: rgba(0,0,0,0) !important;
-  font-size: 1px !important;
-  padding: 0 !important;
-  margin: 0 !important;
-  line-height: normal !important;
-  z-index: 1;
+// Use to hide content visually while keeping it accessible to assistive technologies
+.visually-hidden {  
+  display: block;  
+  position: absolute !important;  
+  width: 1px !important;  
+  height: 1px !important;  
+  padding: 0 !important;  
+  margin: -1px !important;  
+  overflow: hidden !important;  
+  clip: rect(0, 0, 0, 0) !important;  
+  white-space: nowrap !important;  
+  border: 0 !important;  
 }
 
-// Force aria-labels to hide on hidden
-// --------------------------------------------------
-.aria-label.aria-hidden {
-  display: none !important;
-}
+.aria-label {  
+  .visually-hidden();  
+  pointer-events: none;  
+} 
+
+// Use to only display content when it's focused
+.aria-label-focusable, .visually-hidden-focusable {  
+  &:not(:focus):not(:focus-within) {  
```

Comment:
Having revisited this, we should replace `:focus` with `:focus-visible`. This selector is useful to provide a different focus indicator based on the user's input modality (mouse vs. keyboard). Ref: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible
All other instances of `:focus` in Adapt have previously been [updated](https://github.com/adaptlearning/adapt-contrib-core/issues/231).

Reply:
Would we not want elements with this mixin to always show if they've got focus, regardless of keyboard interactivity?

Reply:
Yes, please ignore more comment above. This should remain as `:focus` (ref: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#focus_vs_focus-visible)